db/cardano-sl-db.cabal

```diff
@@ -28,29 +29,138 @@ library
+                       Pos.DB.Update.Logic.Global
+                       Pos.DB.Update.Logic.Local
+                       Pos.DB.Update.MemState.Functions
+                       Pos.DB.Update.MemState.Types
+                       Pos.DB.Update.Poll.DBPoll
+                       Pos.DB.Update.Poll.Logic
+                       Pos.DB.Update.Poll.PollState
+                       Pos.DB.Update.Poll.Pure
+                       Pos.DB.Update.Poll.Logic.Apply
+                       Pos.DB.Update.Poll.Logic.Base
+                       Pos.DB.Update.Poll.Logic.Normalize
+                       Pos.DB.Update.Poll.Logic.Rollback
+                       Pos.DB.Update.Poll.Logic.Softfork
+                       Pos.DB.Update.Poll.Logic.Version
+
+                       Pos.DB.Block.BListener
+                       Pos.DB.Block.GState.BlockExtra
+                       Pos.DB.Block.GState.SanityCheck
+                       Pos.DB.Block.Load
+                       Pos.DB.Block.Logic.Creation
+                       Pos.DB.Block.Logic.Header
+                       Pos.DB.Block.Logic.Internal
+                       Pos.DB.Block.Logic.Util
+                       Pos.DB.Block.Logic.VAR
+                       Pos.DB.Block.Lrc
+                       Pos.DB.Block.Slog.Context
+                       Pos.DB.Block.Slog.Logic
+
+  build-depends:       aeson
+                     , base
                      , bytestring
                      , cardano-sl-binary
+                     , cardano-sl-block
                      , cardano-sl-core
                      , cardano-sl-crypto
+                     , cardano-sl-delegation
+                     , cardano-sl-lrc
+                     , cardano-sl-ssc
+                     , cardano-sl-txp
+                     , cardano-sl-update
                      , cardano-sl-util
                      , concurrent-extra
                      , conduit
                      , containers
+                     , cryptonite
                      , data-default
                      , directory
+                     , ekg-core
                      , ether >= 0.5
                      , filepath
                      , formatting
                      , lens
+                     , log-warper
+                     , lrucache
                      , memory
+                     , mmorph
                      , mtl
+                     , pipes
+                     , reflection
                      , resourcet
                      , rocksdb-haskell-ng
+                     , safe-exceptions
                      , serokell-util
-                     , formatting
+                     , stm
+                     , tagged
+                     , text
+                     , time-units
                      , transformers
                      , universum
+                     , unliftio
+                     , unordered-containers
 
   default-language:    Haskell2010
 
```

Comment:
For my own context, here are the before/after images of the dependency graph:

# Recent commit on `develop`
![cardano-sl-pkg-deps_58414fb62](https://user-images.githubusercontent.com/1151946/43143976-12cde0a0-8f2a-11e8-97bd-7b1dd637443b.png)

# After this PR
![cardano-sl-pkg-deps_cdec-461](https://user-images.githubusercontent.com/1151946/43143977-12db8192-8f2a-11e8-8e25-6e802a60d85d.png)